consensus/vbft/node_utils.go

```diff
@@ -334,8 +334,14 @@ func getCommitConsensus(commitMsgs []*blockCommitMsg, C int) (uint32, bool) {
 		}
 
 		commitCount[c.BlockProposer] += 1
-		if commitCount[c.BlockProposer] > C {
-			return c.BlockProposer, emptyCommit
+		if commitCount[c.BlockProposer] > C && commitCount[c.BlockProposer] == len(c.EndorsersSig) {
```

Comment:
This function can be improved by counting sigs of proposals in commit msgs.

Reply:
ok